plugins/hosts/linux/cap/os_arch.rb

```diff
@@ -0,0 +1,20 @@
+include Vagrant::Util
+
+module VagrantPlugins
```

Comment:
Ok, I see, you need to further qualify in order to download the right binary. I would think that is provided somewhere, but maybe not.